airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/operation/DatabricksStorageOperation.kt

```diff
@@ -116,11 +106,52 @@ class DatabricksStorageOperation(
         workspaceClient.files().createDirectory(stagingDirectory(streamId, database))
     }
 
-    override fun prepareStage(streamId: StreamId, destinationSyncMode: DestinationSyncMode) {
-        prepareStagingTable(streamId, destinationSyncMode)
+    override fun prepareStage(streamId: StreamId, suffix: String, replace: Boolean) {
```

Comment:
Am I correct that this is never called with `replace = true` outside a test context?

Reply:
Nvm. Via `AbstractStreamOperations::prepareStageForTruncate`